docs/how-to-guides/renkulab/external-storage.rst

```diff
@@ -0,0 +1,208 @@
+hosted on AWS and show how to attach it to a Renku interactive session:
+
+- Navigate to the Settings - Cloud storage session in one of your Renku projects.
+
+- Click on the "Add Cloud Storage" button and select ``S3`` for the "Storage Type",
+  then pick "AWS" on the provider list. Click on the "Next" button at the bottom.
+
+  .. image:: ../../_static/images/cloud_storage_2.png
+    :align: center
+    :alt: Storage type and provider selection
+
+
+- Fill in the form by using ``giab`` as the Source path, and
+  ``http://s3.amazonaws.com`` as the Endpoint. We don't need any credentials, and
+  the region is optional. Click on the "Next" button.
+
+  .. image:: ../../_static/images/cloud_storage_3.png
+    :align: center
+    :alt: GIAB bucket options configuration
+
+
+- On the last page, pick any name for the storage (E.G: ``Giab``), than mark
+  this as Read-only. You can change the Mount point too; that determines the
+  virtual folder where this is mounted in the running sessions.
+  Click on the "Add storage" button.
+
+  .. image:: ../../_static/images/cloud_storage_4.png
+    :align: center
+    :alt: GIAB bucket final steps
+
+
+- You can now start a new session and access the bucket at the mount point.
+
+
+Azure Blob example
+~~~~~~~~~~~~~~~~~~
+
+Here is an example of attaching an Azure Blob storage to your project.
+We host some data files in the folder ‘test-data’
+hosted on Azure and show how to attach it to a Renku interactive session:
+
+- Navigate to the Settings - Cloud storage session in your project.
+
+- Click on the "Add Cloud Storage" button and select ``azureblob`` for the
+  "Storage Type". Click on the "Next" button at the bottom.
+
+  .. image:: ../../_static/images/cloud_storage_5.png
+    :align: center
+    :alt: Azure blob storage selection
+
+- Fill in the form by filling the Source path to mount (keep it blank to 
+  mount the default root, or specify the folder) and the Account Name. Fill
+  in the Shared Key.  Click on the "Next" button.
+
+  .. image:: ../../_static/images/cloud_storage_6.png
+    :align: center
+    :alt: Azure blob options configuration
+
+- On the last page, pick any name for the storage (e.g: `blob_storage``), then mark
+  this as Read-only. You can change the Mount point too; that determines the
+  virtual folder where this is mounted in the running sessions.
+  Click on the "Add storage" button.
+
+  .. image:: ../../_static/images/cloud_storage_7.png
+    :align: center
+    :alt: Azure blob final steps
+
+-  You can now start a new session and access this bucket at the mount point.
+
+
+
+WebDAV example
+~~~~~~~~~~~~~~
+
+
+In some cases, you may need to access your cloud storage using a token. In this
+example, we show you how to proceed in such cases for WebDAV storage technology.
+
+We select data file which is 
+hosted on Poly box (ETHZ storage system based on WebDAV) and show how to attach it to 
+a Renku interactive session:
```

Comment:
```suggestion
Many storage services, offer access via the WebDAV protocol. Renku supports mounting WebDAV resources in interactive sessions. 

.. warning:
    We strongly encourage you to generate a token for accessing your private WebDAV shares to avoid sharing your account password.

Here we show an example of adding an `ETH Polybox <https://polybox.ethz.ch>_` drive to the session, but the procedure will be similar for any storage accessible via WebDAV. 
```